pkg/api/getambassador.io/v2/host_types.go

```diff
@@ -94,7 +94,7 @@ type HostSpec struct {
 	// Hostname by which the Ambassador can be reached.
 	Hostname string `json:"hostname,omitempty"`
 
-	// Selector by which we can find further configuration. Defaults to hostname=$hostname
```

Comment:
Nope, it defaults to empty. The Edge Stack ACME client does the wrong thing currently, but it's about to get fixed.